resources/event-hackathon/notebook-authoring-checklist.md

```diff
@@ -0,0 +1,120 @@
+
+## Text
+
+- [ ] All link text is descriptive. It tells users where they will be taken if they open the link.
+- [ ] Use plain language wherever possible.
+- [ ] All acronyms are defined at least the first time they are used. 
+- [ ] Field-specific/specialized terms are used when needed, but not excessively.
+- [ ] Text is broken into paragraphs and/or cells where relevant.
+- [ ] Text is in complete sentences where relevant.
+
+## Code
+
+- [ ] Code sections are introduced/explained before they appear in the notebook. This could be a heading in a prior Markdown cell, a sentence preceding it, or a code comment in the code section.
+- [ ] Code has explanatory comments (if relevant). This is most important for long sections of code.
+- [ ] If the author has control over the syntax highlighting theme in the notebook, that theme has enough color contrast to be legible.
+- [ ] Code and code explanations focus on one task at a time. Unless comparison is the point of the notebook, only one way to complete the task is described at a time.
+
+## Other content
+
+This list is not exhaustive. If you are reviewing a notebook with content that you do not think fits any of these categories, keep in mind
+
+- Text is flexible. Whether it is in the document or linked out, text can be read visually, be read audibly, be magnified, or be translated to another language. Having a text alternative is a good back up plan.
+- Having enough color contrast is required on almost all visual content.
+
+### Videos
+
+- [ ] All videos have titles in the player or in the text before them.
+- [ ] All videos have captions/subtitles. This can include visual information descriptions if relevant.
+- [ ] All videos have transcripts. This can include visual information descriptions if relevant.
+- [ ] All video players have buttons with labels. This can be a persistent label or appear when hovered.
+- [ ] All video players have buttons with enough color contrast.
+- [ ] No videos have flashing images at more than three frames per second.
+
+### Sonifications
+
+- [ ] All audio players have buttons with labels. This can be a persistent label or appear when hovered.
+- [ ] All audio players have buttons with enough color contrast.
+
+### Interactive Widgets 
+
+The accessibility of interactive widgets varies greatly depending how they are included in the notebook. Review beyond this checklist may be needed.
+
+- [ ] All interactive widgets with visual controls have labels. This can be a persistent label or appear when hovered.
+- [ ]  All interactive widgets with visual controls have enough color contrast.
+- [ ] All interactive widgets have a summary of what they are and what they do in a surrounding text area.
+- [ ] If an interactive widget's contents are needed to understand the rest of the notebook, the widget either needs to be tested further or have that content fully represented not as a widget elsewhere in the notebook.
```

Comment:
Can we say something like:

The interactive widget has been usability tested and is confirmed to be accessible.

If accessibility is not confirmed, the content inside the widget is fully represented in a non-widget format elsewhere in the notebook.